index.js

```diff
@@ -1,4 +1,4 @@
 'use strict';
 module.exports = function (n) {
-	return toString.call(n) === '[object Number]' && n > 0;
+	return (typeof n === 'number' || n instanceof Number) && n > 0;
```

Comment:
the reason people do toString is that `instanceof Foo` won't work on a new global object (iframe etc)